helper/resource/testing_test.go

```diff
@@ -71,9 +74,8 @@ func TestTest_factoryError(t *testing.T) {
 		})
 	}()
 
-	fatalStr := fmt.Sprint(mt.FatalArgs)
-	if !strings.Contains(fatalStr, resourceFactoryError.Error()) {
-		t.Fatalf("test should've failed with %s, failed with %s", resourceFactoryError, fatalStr)
```

Comment:
I hate this test in general I think it's pointless and actually prefer delete it, but aside from that I wanted to make sure the code Fatal'd at the place we expected it to, and not for another reason.

Reply:
I see mockT no longer captures the information it once did.